requirements.txt

```diff
@@ -32,3 +32,12 @@ snowballstemmer==1.2.0
 traitlets==4.0.0
 websocket-client==0.34.0
 wsgiref==0.1.2
+
```

Comment:
Do you mean a way of ensuring that the sma-wideband/adc_tests git repo is checked out to a particular commit?  You can just run `git checkout [commit]`, I think.